Plugins/Flow.Launcher.Plugin.Program/Languages/en.xaml

```diff
@@ -40,7 +40,12 @@
     <system:String x:Key="flowlauncher_plugin_program_plugin_description">Search programs in Flow Launcher</system:String>
 
     <system:String x:Key="flowlauncher_plugin_program_invalid_path">Invalid Path</system:String>
-    
+
+    <system:String x:Key="flowlauncher_plugin_program_customizedexplorer">Customized Explorer</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_args">Args</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">You can customized the explorer used for opening the container folder by inputing the Environmental Variable of the explorer you want to use. It will be useful to use CMD to test whether the Environmental Variable is avaliable.</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_args">Enter the customized args you want to add for your customized explorer. %s for parent directory, %f for full path (which only works for win32). Check the explorer's website for details.</system:String>
```

Comment:
sorry i am not too across how the %s and %f option can be used. Could you give me an example?

Reply:
%s is the parent directory and %f is the file path (.exe, .lnk, etc.) I get the idea from the everything plugin which can select the file while opening the parent directory. However, that seems may work not good for uwp.

Reply:
oh ok so this option allows for win32 programs to open both file and parent directory when using 'open containing folder' ?

Reply:
Yes, let me think how I can fix that. If I can't, I will remove that